### PR TITLE
tested code

### PR DESCRIPTION
**Pull request by davidetome**
Added:

- README.md
- scr folder

### CODE REVIEW

File: log-joint/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Log joint data
+The snippet code allow to move a joint between two set points both in position and pwm control mode and logs following joint data:
```
Comment (pattacini) [inline log-joint/README.md]:
```suggestion
The snippet code allows moving a joint between two set points both in position and PWM control mode and logs the following joint data:
```

File: log-joint/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+```console
+cd src
+mkdir build
+cd build
+ccmake ..
+```
+
+## Usage
+Is possible to run the code to control the joint both in `position` and `pwm` control mode by passing to it proper parameters via CLI.
```
Comment (pattacini) [inline log-joint/README.md]:
```suggestion
It is possible to run the code to control the joint both in `position` and `PWM` control mode by passing to it proper parameters via CLI.
```

**Review by pattacini — CHANGES_REQUESTED** [review]
Nicely done!
Only minor fixes you may integrate straight away.

**Review by pattacini — APPROVED** [review]

**Comment by pattacini** [comment]
Hi @arrenglover

With this PR, @davidetome made available a testing suite we aim to use for verifying the performance of the `iCubGenova02` current head as I told you early on.

The code is documented at https://github.com/icub-tech-iit/study-improve-head-edpr/tree/code/log-joint.

@davidetome will be off next week but back to work soon afterward to assist you with the preliminary tests.

**Comment by arrenglover** [comment]
okay great! thanks @davidetome and @pattacini, that sounds good to me.

**Pull request merged**